### PR TITLE
Improve and update eventlog README

### PR DESCRIPTION
**Pull request by kc1212**
More documentation is always good and remove SecureKG reference.

### CODE REVIEW

File: eventlog/README.md [hunk 1]
```diff
@@ -22,14 +22,17 @@ For the general information about running a conode, or conodes, please see the
...
 ## Client API
-We offer three ways for clients to connect to the event log service.
+We offer three ways for clients to connect to the event log service. All the
+APIs expect an existing OmniLedger object that has a darc with "spawn:eventlog"
+and "invoke:eventlog" in its rules. The eventlog signer (which we will create
+below) *must* be authorised to use these rules. For testing SecureKG, this step
+is not needed, please see [below](#SecureKG).
 
 ### Go API
+*The go eventlog API is subject to change because it will create an omniledger
```
Comment (jeffallen) [inline eventlog/README.md]: This is fixed in #1314 .

File: eventlog/README.md [hunk 2]
```diff
@@ -49,22 +52,54 @@ The detailed API can be found on
...
+#### SecureKG
+For testing SecureKG, you may use the information in the
+[`SecureKG`](https://github.com/dedis/cothority/blob/master/external/java/src/main/java/ch/epfl/dedis/lib/omniledger/SecureKG.java)
+class to initialise OmniLedger and EventLogInstance. Here is an example.
+```java
+OmniLedgerRPC ol = SecureKG.getOmniledgerRPC();
+Signer admin = SecureKG.getSigner();
+EventLogInstance el = new EventLogInstance(ol, SecureKG.getEventlogId());
+InstanceId key = el.log(new Event("hello", "goodbye"), Arrays.asList(admin))
+el.get(key)
```
Comment (jeffallen) [inline eventlog/README.md]: will this work? Seems like the get will fail until the transaction is run?

File: eventlog/README.md [hunk 3]
```diff
@@ -49,22 +52,54 @@ The detailed API can be found on
...
 ```
 
+We also have a search API, which allows searching for a particular topic within
+a time-range.
+```java
+long now = System.currentTimeMillis() * 1000 * 1000;
+SearchResponse resp = el.search("", now - 1000, now + 1000);
+```
+
+Please refer to the javadocs for more information. The javadocs are not hosted
```
Comment (jeffallen) [inline eventlog/README.md]: let's fix this... put them on cothority.dedis.ch/java maybe?

**Comment by ineiti** [comment]
need to separate the securekg doc - that's not to be in master for the moment.

**Comment by ineiti** [comment]
Wonderful!

**Pull request merged**